package.json

```diff
@@ -456,9 +456,45 @@
             "end"
           ]
         },
+        "mssql.format.alignColumnDefinitionsInColumns": {
+          "type": "boolean",
+          "description": "Should column definitions be aligned?",
```

Comment:
are these settings optional? I think the convention we were using before is to prepend the description with [Optional] if they are.